src/use/use.py

```diff
@@ -314,23 +330,20 @@ def __del__(self):
         self.stop()
         atexit.unregister(self.stop)
 
-class Use:
+class Use(ModuleType):
+    from pathlib import Path
```

Comment:
I had to think twice before I saw what you did there :) Please add a comment with an explanation

Reply:
Commented in the next update

Reply:
ahh.. that wasn't was I commented on - i meant why the imports had to be in the class, not in the file header